src/e2e/emulated-devices.js

```diff
@@ -0,0 +1,16 @@
+import { devices } from 'puppeteer'
+
+export const iPhone = devices['iPhone 11']
+
+export const desktop = {
```

Comment:
Why emulate desktop? Can we use puppeteer without emulation in this case?

Reply:
If we set the emulator once for any test, we need to set the emulator again for another platform. Puppeteer does not have any default value or function like a reset for desktop.

Reply:
Is the page not destroyed after each test? Seems like we would like full isolation between tests so I’m not sure how it was set up. 

If not, we could reset to defaults in `afterEach`. `page.emulate` is an alias for `page.setUserAgent` which has defaults, and `page.setViewport`, which may be able to be set to an empty string or a basic Chrome string. 

https://pptr.dev/#?product=Puppeteer&version=v9.0.0&show=api-pageemulateoptions

Reply:
No, the page is not destroyed after each test. We only reset DB and local storage as you can see here:
https://github.com/cybersemics/em/blob/5474052ae8e9d5f306f9f8afc5a8282348a9fe8e/src/e2e/__tests__/caret.ts#L16-L21 

I will look at how we can achieve isolation between tests. The way it is doesn't give me much confidence too.

Maybe, we can pass an empty string to `page.setUserAgent` but I think [page.setViewPort](https://pptr.dev/#?product=Puppeteer&version=v9.0.0&show=api-pagesetviewportviewport)
 doesn't get string. We need to pass a Viewport object.

Reply:
> I will look at how we can achieve isolation between tests. The way it is doesn't give me much confidence too.

Great. I think that will be better.

> Maybe, we can pass an empty string to `page.setUserAgent` but I think [page.setViewPort](https://pptr.dev/#?product=Puppeteer&version=v9.0.0&show=api-pagesetviewportviewport)
> doesn't get string. We need to pass a Viewport object.

Sorry! I meant the other way around. `page.emulate` is an alias for `page.setViewport` which has defaults (except dimensions), and `page.setUserAgent`, which may be able to be set to an empty string or a basic Chrome string.